addons/pos_restaurant/static/src/js/Screens/ProductScreen/ControlButtons/TableGuestsButton.js

```diff
@@ -26,7 +27,21 @@ odoo.define('pos_restaurant.TableGuestsButton', function(require) {
             });
 
             if (confirmed) {
-                this.env.pos.get_order().set_customer_count(parseInt(inputNumber, 10) || 1);
+                const guestCount = parseInt(inputNumber, 10);
```

Comment:
If it's empty, it should still be be counted as 1 
```suggestion
                const guestCount = parseInt(inputNumber, 10) || 1;
```

Reply:
Done